kafka/conn.py

```diff
@@ -862,7 +873,7 @@ def connection_delay(self):
         large number to handle slow/stalled connections.
         """
         if self.disconnected() or self.connecting():
-            if len(self._gai) > 0:
+            if len(self._gai) > 0 or self._socks5_proxy.use_remote_lookup():
```

Comment:
won't this crash if/when self._socks5_proxy is None and self._gai is empty?

Reply:
You're completely right. I added some extra logic now to check whether `self._socks5_proxy` is set.